package.json

```diff
@@ -14,10 +14,9 @@
     "axios": "^0.18.1",
     "core-js": "^3.6.4",
     "css-loader": "^0.28.11",
-    "launchdarkly-js-client-sdk": "^2.17.0",
```

Comment:
This was a recent fix that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit